includes/class-move.php

```diff
@@ -49,4 +51,40 @@ public static function extend_actor_profiles( $actor, $id, $user ) {
 
 		return $actor;
 	}
+
+	/**
+	 * Move an ActivityPub account from one location to another.
+	 *
+	 * @param string $from The current account URL.
+	 * @param string $to   The new account URL.
+	 * @return int|bool The ID of the outbox item or false on failure.
+	 */
+	public static function account( $from, $to ) {
+		$user = Actors::get_by_resource( $from );
+
+		if ( ! $user || is_wp_error( $user ) ) {
+			return false;
+		}
+
+		// Update the movedTo property.
+		\update_user_option( $user->get__id(), 'activitypub_move_to', $to );
+
+		// Add the old account URL to alsoKnownAs.
+		$also_known_as   = (array) \get_user_option( 'activitypub_also_known_as', $user->get__id() );
+		$also_known_as[] = $from;
+		$also_known_as   = array_unique( $also_known_as );
+		\update_user_option( $user->get__id(), 'activitypub_also_known_as', $also_known_as );
+
+		// Create Move activity.
+		$activity = array(
```

Comment:
we have a problem! if we use it this way, the outbox handler would set the activity as object!

This way it would look like:

```JSON
{
	"type": "Move",
	"object": {
		"type": "Move",
		"actor": "...",
		"object": "...",
		"target": "..."
	}
}
```